### PR TITLE
Fix raising filenotfounderror

### LINKED ISSUE #43: `FileNotFoundError` not raised correctly
@paulmueller If I am not wrong, the `FileNotFoundError` and its variant are not raised correctly. `errno.ENOENT` and `filename` are missing here:

https://github.com/GuckLab/impose/blob/5b2907bbb253ea968eba47bf86eeaf5f121dac00/impose/data.py#L87
https://github.com/GuckLab/impose/blob/5b2907bbb253ea968eba47bf86eeaf5f121dac00/impose/session.py#L279-L284

### PR DESCRIPTION
**Pull request by raimund-schluessler**
This now hopefully correctly raises the `FileNotFoundError`.

Closes #43.

### CODE REVIEW

File: impose/session.py [hunk 1]
```diff
@@ -278,9 +279,11 @@ def find_file(name, search_paths, signature=None):
         else:
             raise ImposeDataFileNotFoundError(
                 name,
```
Comment (paulmueller) [inline impose/session.py:281]: This line should be removed, correct?

**Review by paulmueller — CHANGES_REQUESTED** [review]
Thanks for spotting this!

File: impose/session.py (hunk 1 above)
Comment (raimund-schluessler) [reply impose/session.py:281]: I think so, yes. Just wasn't sure whether I should touch it or not. I will adjust the PR.

Comment (raimund-schluessler) [reply impose/session.py]: > I think so, yes. Just wasn't sure whether I should touch it or not. I will adjust the PR.

Done.

**Pull request merged**